### PR TITLE
Feature/z cookiebar

### PR DESCRIPTION
**Pull request by ufollettu**
# Component - Z-Cookiebar - Cookiebar Pattern Component

----

### Motivation and Context

This PR adds a cookiebar pattern to handle cookie policy info
 
### Priority

- [ ] 1 - Highest
- [ ] 2 - High
- [X] 3 - Medium
- [ ] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [X] Component (add a Component as approved dy Design System)
- [ ] Docs (add documentation)
- [ ] Chore (changes that adds small enhancement)
- [ ] Breaking (fix or feature that would cause existing functionality to not work as expected)

### Abstract and Screenshots

https://app.abstract.com/projects/fd370780-e659-11e8-99dc-0d08537c5fde/branches/master/commits/latest/files/7e5534c1-96e2-44c2-98f8-965c34ee8835/layers/BF4A79C8-0BD8-42E0-A899-51F708D1526E?collectionId=ecb94390-fedf-428f-b021-4d452c356519&collectionLayerId=2a417638-205d-43b0-b7d7-be80cbcd3b88&mode=build&selected=root-C23B113F-BEB1-4805-8EC6-C9771778E28A

### Note

----

## Component and Fix approval flow to Master branch

A Pull Request to be merged must have two approvals, one from a member of the product team who developed it and another from a member of the dst dev team other than the team representative.

### CODE REVIEW

File: src/components/notification/z-cookiebar/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    expect(page.root).toEqualHtml(`
+    <z-cookiebar
+      cookiepolicyurl="https://my.zanichelli.it/privacy#Cookie"
+      hide="false">
+        <mock:shadow-root>
+          <z-candybar class="">
+            <div class="content" slot="content">
+              <div>
+                <h2>Su questo sito usiamo i cookie.</h2>
```
Comment (aliGamb) [inline src/components/notification/z-cookiebar/index.spec.ts:23]: Il testo non va passato tramite prop (in previsione della parte di traduzione) ?

File: src/components/notification/z-cookiebar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        this.callback();
+      }
+    }
+  }
+
+  renderContentSlot() {
+    return (
+      <div class="content" slot="content">
+        <div>
+          <h2>Su questo sito usiamo i cookie.</h2>
```
Comment (aliGamb) [inline src/components/notification/z-cookiebar/index.tsx:49]: n previsione della parte di traduzione

File: src/components/notification/z-cookiebar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  /** emitted on ACCETTA button click, returns event */
+  @Event() accept: EventEmitter;
+  emitAccept(ev: MouseEvent | KeyboardEvent) {
+    this.accept.emit({ev});
+  }
+
+  handleOkButtonClick(ev: MouseEvent) {
+    this.emitAccept(ev);
+    if (
+      typeof this.callback !== "undefined" &&
```
Comment (aliGamb) [inline src/components/notification/z-cookiebar/index.tsx:26]: È necessario il check sul !== "undefined" visto che poi viene aggiunta la condizione === "function"?

File: src/components/notification/z-cookiebar/index.tsx (hunk 2 above)
Comment (ufollettu) [reply src/components/notification/z-cookiebar/index.tsx:49]: right now there are no texts in languages ​​other than Italian

File: src/components/notification/z-cookiebar/index.tsx (hunk 3 above)
Comment (ufollettu) [reply src/components/notification/z-cookiebar/index.tsx]: fixed

Comment (ufollettu) [reply src/components/notification/z-cookiebar/index.tsx]: fixed

**Review by vpasquino — APPROVED** [review]

File: src/components/notification/z-cookiebar/index.tsx (hunk 2 above)
Comment (aliGamb) [reply src/components/notification/z-cookiebar/index.tsx:48]: ok :)

**Review by aliGamb — APPROVED** [review]

**Review by a-sartori-wt — APPROVED** [review]

**Pull request merged**